model/src/main/java/com/basistech/rosette/apimodel/CategoriesOptions.java

```diff
@@ -31,8 +31,26 @@
 public class CategoriesOptions extends Options {
 
     /**
+     * @deprecated
+     * Users should use {@link singleLabel} to return only one result
+     * or {@link scoreThreshold} to filter results based on raw score
      * @return number of categories
      */
     @Min(1)
     private final Integer numCategories;
+
+    /**
+     * Single label mode will return only the highest scoring category
+     * If singleLabel is false, every category whose score exceeds the default
+     * (or specified) {@link scoreThreshold} value will be returned
+     * @return whether or not we are in single label mode
+     */
+    private final Boolean singleLabel;
+
+    /**
+     * threshold against the category's raw score, whose value
+     * can be any real number.
+     * @return the score threshold
+     */
+    private final Float scoreThreshold;
```

Comment:
maybe worthwhile to clarify that the threshold will be ignored if `singleLabel` is `true`, especially if the single label score falls below the threshold? or did I interpret it wrong?

Reply:
turning on `singleLabel` automatically sets `scoreThreshold` to -INF, but this will get overridden if `scoreThreshold` is also specified.  I'll add a clarifying note to `singleLabel` that the highest scoring category gets returned regardless of score (although after talking to Hannah we may hold off on exposing the raw score altogether, so this may all be moot)

Reply:
So not merging yet?